src/detect-engine.c

```diff
@@ -958,7 +958,20 @@ static uint32_t DetectBufferTypeHashNameFunc(HashListTable *ht, void *data, uint
     const DetectBufferType *map = (DetectBufferType *)data;
     uint32_t hash = hashlittle_safe(map->name, strlen(map->name), 0);
     hash += hashlittle_safe((uint8_t *)&map->transforms, sizeof(map->transforms), 0);
+    // Add transform data, if any. Note that the transform identifier is always added; if
+    // the transform supports serialization, that data will also be added.
+    for (int i = 0; i < map->transforms.cnt; i++) {
+        const TransformData *t = &map->transforms.transforms[i];
+        hash += hashlittle_safe((uint8_t *)&t->transform, sizeof(t->transform), 0 );
```

Comment:
here we'd only use the xform id now, right? Also, we need to include some positioning info?

Reply:
Will update.